### PR TITLE
Init seamless

### PR DESCRIPTION
**Pull request by oxkatsudon**

### CODE REVIEW

File: pages/404.page.tsx [hunk 1]
```diff
@@ -32,9 +32,6 @@ export default function Aave404Page() {
             backgroundColor: theme.palette.mode === 'dark' ? 'transparent' : '',
           }}
         >
-          <Box sx={{ maxWidth: 444, m: '0 auto' }}>
```
Comment (0xforestwalker) [inline pages/404.page.tsx:35]: Why is this removed?

Comment (oxkatsudon) [reply pages/404.page.tsx:35]: this is the 404 aave ghost image

File: src/components/WalletConnection/WalletSelector.tsx [hunk 2]
```diff
@@ -219,60 +134,6 @@ export const WalletSelector = () => {
         walletName="Coinbase Wallet"
         walletType={WalletType.WALLET_LINK}
       />
-      <WalletRow key="torus_wallet" walletName="Torus" walletType={WalletType.TORUS} />
```
Comment (0xforestwalker) [inline src/components/WalletConnection/WalletSelector.tsx:222]: Can we keep this?

File: src/ui-config/networksConfig.ts [hunk 3]
```diff
@@ -263,6 +263,27 @@ export const networkConfigs: Record<string, BaseNetworkConfig> = {
     },
     // ratesHistoryApiUrl: 'https://aave-api-v2.aave.com/data/rates-history',
   },
+  [84531 as ChainId /*ChainId.base_goerli*/]: {
+    name: 'Base Görli',
+    publicJsonRPCUrl: [
+      'https://base-goerli.gateway.tenderly.co/1SVyxsO1IHL8NHPgMyQmuF',
+      // 'https://goerli.base.org',
```
Comment (0xforestwalker) [inline src/ui-config/networksConfig.ts:270]: Can we uncomment the public RPC as well?

File: src/ui-config/marketsConfig.tsx [hunk 4]
```diff
@@ -79,231 +53,27 @@ export type MarketDataType = {
...
-  proto_fuji = 'proto_fuji',
-  proto_polygon = 'proto_polygon',
-  proto_mumbai = 'proto_mumbai',
-  amm_mainnet = 'amm_mainnet',
-  proto_goerli = 'proto_goerli',
-  // external
-  // permissioned_market = 'permissioned_market',
 }
 
+const BASE_ADDRESSES = {
```
Comment (0xforestwalker) [inline src/ui-config/marketsConfig.tsx:62]: These are testnet addresses only

File: src/ui-config/marketsConfig.tsx [hunk 5]
```diff
@@ -314,363 +84,40 @@ export const marketsData: {
...
-      COLLECTOR: AaveV3Metis.COLLECTOR,
-    },
-    halIntegration: {
-      URL: 'https://app.hal.xyz/recipes/aave-v3-track-health-factor',
-      marketName: 'polygon',
+      LENDING_POOL_ADDRESS_PROVIDER: BASE_ADDRESSES.LENDING_POOL_ADDRESS_PROVIDER,
+      LENDING_POOL: BASE_ADDRESSES.LENDING_POOL,
+      WETH_GATEWAY: BASE_ADDRESSES.WETH_GATEWAY,
+      WALLET_BALANCE_PROVIDER: BASE_ADDRESSES.WALLET_BALANCE_PROVIDER,
+      //WALLET_BALANCE_PROVIDER: AaveV3Base.WALLET_BALANCE_PROVIDER,
```
Comment (0xforestwalker) [inline src/ui-config/marketsConfig.tsx:115]: We can remove this
